wildfire1/lib/ui/views/reportafire.dart

```diff
@@ -0,0 +1,459 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_form_bloc/flutter_form_bloc.dart';
+
+class FireReport extends StatefulWidget {
+  const FireReport({Key? key}) : super(key: key);
+
+  @override
+  _FireReportState createState() => _FireReportState();
+}
+
+var timestamp = new DateTime.now();
+
+class _FireReportState extends State<FireReport> {
+  TextEditingController detailsController = TextEditingController();
+  TextEditingController firstNameController = TextEditingController();
+  TextEditingController lastNameController = TextEditingController();
+  TextEditingController phoneNumberController = TextEditingController();
+  TextEditingController locationController = TextEditingController();
```

Comment:
Instead of TextController, defing a variable for each Field depending on there datatypes
eg String lastname